abakus-statutter/08-generalforsamling.tex

```diff
@@ -53,7 +53,7 @@ \subsubsection{}
 alminnelig flertall.
 
 \subsubsection{}
-Vedtak fattet av en generalforsamling kan bare annulleres av en generalforsamling.
+Vedtak fattet av en generalforsamling kan bare annulleres av en Generalforsamling.
```

Comment:
The first 'generalforsamling' needs an uppercase G

Reply:
Nope, and the last "generalforsamling" should have also be a lowercase "G"

Reply:
`en generalforsamling` snakker vel om "en generell" generalforsamling, ikke selve organet?

Reply:
Endelig en som snakker norsk! Ja, stemmer. Derfor skal det være liten "G"

Reply:
Men, ble denne endringen også vedtatt med zoom-polls når jeg ikke fulgte med, eller er det en separat redaksjonell endring? Om det er sistnevnte burde den vel ikke merges her, men heller som et konkret forslag som HS kan ta stilling til separat?

Reply:
Sistnevnte. Gjelder forøvrig også de andre hvor det er omgjort fra liten til stor "G".

Er litt usikker her, men mulig du har svaret. HS kan gjøre redaksjonelle endringer så lenge de offentliggjøres 2. april eller 1. oktober. Så da er det vel lov å merge de sammen nå, så lenge HS offentliggjør dem 2. april?

Reply:
> Endelig en som snakker norsk! Ja, stemmer. Derfor skal det være liten "G"

Hahaha. Synes det ryddigst å snakke norsk når endringene vi diskuterer er _på norsk_ de også... :sweat_smile:

Reply:
> Sistnevnte. Gjelder forøvrig også de andre hvor det er omgjort fra liten til stor "G".

Nice!

> Er litt usikker her, men mulig du har svaret. HS kan gjøre redaksjonelle endringer så lenge de offentliggjøres 2. april eller 1. oktober. Så da er det vel lov å merge de sammen nå, så lenge HS offentliggjør dem 2. april?

Jaaa.

Men ref § 8.3.4 kan ikke endringene tre i kraft før tidligst 15. april, og antar de tenker å sende de nye statuttene til Brønnøysund før den tid...

> § 8.3.4 for ref:
> Hovedstyret har til enhver tid myndighet til å gjøre redaksjonelle endringer i Abakus sine statutter. Eventuelle endringsforslag må offentliggjøres 2. april eller 1. oktober.
> 
> Hovedstyret må redegjøre for de endringer de har gjort ved å offentliggjøre disse på foreningens nettside. Dersom et medlem av Abakus gir skriftlig uttrykk til Hovedstyret innen to -2- uker om at en endring ikke ivaretar statuttens opprinnelige intensjon, må endringen behandles på generalforsamling. Endringer som ikke blir disputert i løpet av denne perioden trer i kraft to -2- uker etter offentliggjøringen.

Reply:
Her har du helt rett! Leste ikke andre delen av statutten 😅

Reply:
Tanken var som @PravKiru nevnte, så tenkte å få HS til å godkjenne de små endringene før merge. Men leste nok ikke godt nok jeg heller. Kan fjerne de og legge den i en annen PR som kan merges etter dette.

Reply:
Smekk det sammen med `Fondsstyret` og `digital sikkerhet` a

Reply:
Smart! Will do :shipit: